### DISCUSSION TITLE
geodcat-ap/dcat2 as a replacement for iso19139?

### DISCUSSION THREAD

**Discussion by pvgenuchten** [INSPIRE-MIF/helpdesk#79] [Ideas]
I was looking at page https://inspire.ec.europa.eu/good-practice/geodcat-ap

In my opinion the page should better highlight what is the actual good practice endorsed on this page. From a first impression some may expect that the good practice enables capabilities to publish a metadata record in either format to fit with INSPIRE regulation. However the page seem to (only) endorse to publish a metadata record in iso19139 as well as in geodcat-ap.

I'm curious to learn if there are initiatives to extend the good practice, to enable also the first case (publish in either format). Technology as part of a metadata catalogue or geoportal would be enabled for ingestion of either iso19139 or geodcat-ap, validate the document against a set of validation rules and expose it in either format as requested. To me such an initiative would fit in a new generation of INSPIRE using more common ontologies and formats (json-ld). An interesting case here would be to describe how one could extend the OGCAPI-Records [Record format](https://github.com/opengeospatial/ogcapi-records/blob/master/core/openapi/schemas/recordGeoJSON.yaml) to include the properties required by INSPIRE.

Additional side note is that the hyperlinks to https://joinup.ec.europa.eu/document/geodcat-ap-implementations and https://inspire.ec.europa.eu/pr_searchx.cfm?i=5&id_search=50190 are currently broken.

**Comment by sMorrone** [comment]
@hernlor or @MarcoMinghini could you please support on issue related to broken hyperlinks? Thank you

**Comment by MarcoMinghini** [comment]
Dear @pvgenuchten and @sMorrone,
the links in the Good Practice page were fixed.

**Comment by sMorrone** [comment]
@hylkevds 
know you're working on an INSPIRE compliant metadata system in the Gaia-X context - could you provide more information? Could this be linked to @pvgenuchten 's point "_Technology as part of a metadata catalogue or geoportal would be enabled for ingestion of either iso19139 or geodcat-ap, validate the document against a set of validation rules and expose it in either format as requested. To me such an initiative would fit in a new generation of INSPIRE using more common ontologies and formats (json-ld)_"

**Comment by sMorrone** [comment]
I am moving this issue to the **Discussions**.
